scripts/helm-init.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
 set -exou pipefail
 
-CLUSTER=$1
-WD=$(pwd)
-kubectl create serviceaccount --namespace kube-system tiller --kubeconfig=${CLUSTER}".conf" || true
-kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller --kubeconfig=${CLUSTER}".conf" || true
+CLUSTERCONF=$1
+kubectl create serviceaccount --namespace kube-system tiller --kubeconfig=${CLUSTERCONF} || true
```

Comment:
What's the purpose of `|| true` in these lines? If this command fails for whatever reason, the subsequent lines would also fail wouldn't they?

Reply:
There's no real purpose, other than them already being there. Want me to remove them?

Reply:
I think as is this kind of defeats the purpose of `set -e`